migrating-v2-v3.md

```diff
@@ -0,0 +1,109 @@
+# Migrating FingerprintJS v2 to v3
+
+This guide shows how to migrate your code from FingerprintJS version 2 to version 3.
+
+## Installation
+
+The migration process depends on the way you install the library.
+
+### CDN
+
+Use the new URL: https://cdn.jsdelivr.net/npm/@fingerprintjs/fingerprintjs@3/dist/fp.min.js
+
+### Bower
+
+Bower support is dropped, choose another installation method.
+
+### NPM or Yarn
+
+Update the package version:
+
+```bash
+npm install @fingerprintjs/fingerprintjs@^3
```

Comment:
why do we need to specify a version explicitly? Will just using `npm install` not work?

Reply:
It will until we release v4. It’s not a big problem actually, just `npm install` will be fine.